device/common/include/traccc/clusterization/device/ccl_kernel.hpp

```diff
@@ -87,7 +88,7 @@ struct ccl_kernel_helper {
 /// put into
 template <typename barrier_t>
 TRACCC_DEVICE inline void ccl_kernel(
-    details::index_t threadId, details::index_t blckDim, unsigned int blockId,
+    const thread_id1 thread_id,
```

Comment:
Is it better to pass this object by value, or by reference? I'm not actually sure...